### PR TITLE
add scoped variables

### PR DESCRIPTION
**Pull request by chewnoill**
Hey! 👋 

This PR brings `next-mdx-remote` more in line with how the mdx-runtime works.

see: https://github.com/mdx-js/mdx/blob/master/packages/runtime/src/index.js

Great work over here by the way, I like what you're doing

### CODE REVIEW

**Comment by jescalan** [comment]
This is incredible, thank you so much for the contribution @chewnoill! Going to take a moment to ensure I fully understand the changes that were made here, then will get this shipped out :shipit:

File: render-to-string.js [hunk 1]
```diff
@@ -1,24 +1,16 @@
...
   // transform it into react
-  const renderer = `
-import React from 'react'
-import { mdx } from '@mdx-js/react'
-`
-  return mdx(source, options)
-    .then((jsx) => {
-      return `${renderer}\n${jsx}`
-    })
+  return mdx(source, { ...options, skipExport: true })
```
Comment (jescalan) [inline render-to-string.js:13]: I hadn't dug up this `skipExport` option, this is a great find. I think we can also remove [this transform](https://github.com/hashicorp/next-mdx-remote/blob/master/babel-plugin-mdx-browser.js#L28) safely now that it has been added!

Comment (chewnoill) [reply render-to-string.js:13]: 👍

**Comment by jescalan** [comment]
👏🏼 thank you again for the contribution! I'll have a release out for this shortly

**Pull request merged**